current_fall2025/scripts/app.py

```
@@ -141,28 +171,53 @@ def process_message(query: str):
     if not reranked_docs:
         return "No relevant items found after reranking.", []
 
     # --- STEP 5: Summarization ---
     context_text = "\n\n".join(d.page_content for d in reranked_docs if d.page_content)
     summary = generate_catalog_summary(llm, expanded_query, context_text)
-    
+
     return summary, reranked_docs
 
+
+def display_error_with_retry(error_message: str, query: str):
```

Ruff (0.14.8)

Unused function argument: `query`

(ARG001)